src/code/z_actor.c

```diff
@@ -4420,18 +4420,19 @@ s16 func_800BDB6C(Actor* actor, PlayState* play, s16 arg2, f32 arg3) {
     return arg2;
 }
 
-void Actor_ChangeAnimationByInfo(SkelAnime* skelAnime, AnimationInfo* animationInfo, s32 animIndex) {
-    f32 frameCount;
+void Actor_ChangeAnim(SkelAnime* skelAnime, AnimationInfo* animationInfo, s32 animIndex) {
+    f32 endFrame;
```

Comment:
For Example, is this a function we want to follow the common `_ChangeAnim` or keep the `ByInfo`? There are maybe ~10 specific actor functions that are copy-pastes of this code, so then would those also keep the `ByInfo` in their name?

Reply:
This is a weird situation, since for the general helper functions there are 3 `_ChangeAnim` functions (1 for each type of animation struct) and so need to be distinguished (the other 2 being in SubS). The actors however only need to deal with one animation struct. With that in mind, I suggest leave `ByInfo` here, but the actors have the `ByInfo` removed. (I agree with shortening Animation -> Anim for all three functions though).